ruby-challenges.rb

```diff
@@ -32,3 +46,18 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_test_case3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def is_palindrome (str)
+    temp = str.downcase
+    if temp.reverse == temp
+        puts "#{str} is a palindrome"
+    elsif temp.reverse != temp 
+        puts "#{str} is not a palindrome"
```

Comment:
Make sure you don't leave any `puts` or `p` within your methods. It's essentially the same as leaving in console.logs in Javascript, which is bad practice.